### PR TITLE
DEV-3813-exam-03-roman-numbers-remake-exercise

### PR DESCRIPTION
**Pull request by mikysett**
merge `public` and `rust-tests` at the same time

### CODE REVIEW

**Review by nprimo — COMMENTED** [review]
What should happen when the string to parse the new `OfficeWorker` has more or fewer fields than expected?
What should happen when the string to parse the `WorkerRole` is not correctly formatted - e.g. does not match the cases described in the subject?

**Review by nprimo — CHANGES_REQUESTED** [review]
The subject should specify what happens when there are some wrong format inputs for the `WorkerRoler` fields

**Comment by mikysett** [comment]
I've added a note specifying that invalid inputs won't be tested

File: subjects/office_worker/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+## office_worker
+
+### Instructions
+
+Create a structure `OfficeWorker` having the following public fields:
```
Comment (nprimo) [inline subjects/office_worker/README.md:5]: Running locally `npx prettier --write .` is giving me an extra new line here

**Review by nprimo — CHANGES_REQUESTED** [review]
There is a small format issue with `prettier` where I have added the comment - please, let me know if it's not the same for you

File: subjects/office_worker/README.md (hunk 1 above)
Comment (mikysett) [reply subjects/office_worker/README.md:5]: done

**Review by nprimo — APPROVED** [review]
LGTM!

**Pull request merged**